src/content/docs/en/reference/experimental-flags/csp.mdx

```diff
@@ -411,3 +456,30 @@ After the build, the `<meta>` element for this individual page will add your has
   "
 >
 ```
+
+### `insertFontResource`
+
+<p>
+
+**Type:** `(resource: string) => void`<br />
+<Since v="5.12.7" />
+</p>
+
+Inserts a new resource to be used for the `font-src` directive.
+
+```astro
+---
+Astro.insertFontResource("https://fonts.cdn.example.com");
+---
+```
+
+After the build, the `<meta>` element for this individual page will add your source to the default `style-src` directive:
+
+```html
+<meta
+  http-equiv="content-security-policy"
+  content="
+    font-src https://fonts.cdn.example.com;
```

Comment:
Is this the same situation where doing this overrides and removes `'self'`? If so, feels like that should also be mentioned, since it will remove the default?

Reply:
In this case, it's merged with the config so we could show `'self'` if we wanted. Maybe @ematipico has an opinion on this!

Reply:
OK, so if this merges that's probably easier to document! We would just make sure that we describe it as "additive"